src/command/client/search.rs

```diff
@@ -626,14 +634,14 @@ async fn select_history(
 
         // Handle input
         if let Event::Input(input) = events.next()? {
-            if let Some(output) = key_handler(&input, &mut app) {
+            if let Some(output) = key_handler(exit_mode, &input, &mut app) {
```

Comment:
Hmm. I wonder if we should put this parameter inside the app state or not

Thinking about it, this is fine I think

Reply:
If we're going that route, does it make sense to maybe just add the settings to the app state, so we don't need to add parameters / change app state for every change like this?